lib/user-interface/react/src/components/chatbot/FileUploadModals.tsx

```diff
@@ -85,25 +86,44 @@ export type ContextUploadProps = {
     setShowContextUploadModal: React.Dispatch<React.SetStateAction<boolean>>;
     fileContext: string;
     setFileContext: React.Dispatch<React.SetStateAction<string>>;
+    selectedModel: IModel;
 };
 
 export function ContextUploadModal ({
     showContextUploadModal,
     setShowContextUploadModal,
     fileContext,
     setFileContext,
+    selectedModel
 }: ContextUploadProps) {
     const [selectedFiles, setSelectedFiles] = useState<File[] | undefined>([]);
     const dispatch = useAppDispatch();
     const notificationService = useNotificationService(dispatch);
+    const modelSupportsImages = selectedModel?.features.filter((feature) => feature.name === 'imageInput').length > 0;
```

Comment:
Great user of features! Not necessarily needed here but as we use more and more features we'll probably want to make this into a helper function to clean up the code.